File: lista-1/quest2.java
import java.util.Random;

public class quest2{
    public static void main(String [] args){
      
    System.out.println("Jogo do Craps.");
    int  ponto=0,dado1, dado2, somaD, lancamento = 0;

    Random rDado1 = new Random();
    Random rDado2 = new Random();
      
    dado1 = 1 + rDado1.nextInt(6);
    dado2 =1 + rDado2.nextInt(6);

    somaD = dado1 + dado2;

    System.out.println("Dado 1: " + dado1 + " Dado2: " + dado2 + "\nPonto: " + somaD);
    
    

    if(somaD == 7 || somaD == 11){
      System.out.println("Parabéns, você ganhou!");
      lancamento = 1;
    }else if(somaD ==2 || somaD == 3 || somaD == 12){
        System.out.println("CRAPS! Você perdeu");
        lancamento = 1;
    }else{
        ponto = somaD;
        System.out.println("Iniciando estágio 2");
    }
      
    while(lancamento==0){
     dado1 = 1 + rDado1.nextInt(6);
     dado2 =1 + rDado2.nextInt(6);
     somaD = dado1 + dado2;

     System.out.println("Dado 1: " + dado1 + " Dado2: " + dado2 + "\nPonto: " + somaD);

     if(somaD == 7 ){
        System.out.println("CRAPS! Você perdeu");
        lancamento = 1;
     }
     else if(somaD == ponto){
        System.out.println("Parabéns, você ganhou!");
        lancamento = 1;
     }

    }
 
}
}

File: lista-2/quest2.py
import random

print("JOGO DO CRAPS")

rDado1 = random.randint(1,6)
rDado2 = random.randint(1,6)

somaD = rDado1 + rDado2

print("Dado 1: " + str(rDado1) + " Dado 2: " + str(rDado2))

lancamento = 0

if(somaD == 7 or somaD == 11):
   print("Parabéns, você ganhou!")
   lancamento = 1

elif(somaD ==2 or somaD == 3 or somaD == 12):
    print("CRAPS! Você perdeu")
    lancamento = 1
else:
    ponto = somaD
    print("Iniciando estágio 2")

while(lancamento==0):
     dado1 = random.randint(1,6)
     dado2 = random.randint(1,6)

     somaD = dado1 + dado2

     print("Dado 1: " + str(rDado1) + " Dado 2: " + str(rDado2) + " Ponto: " + str(somaD) )

     if(somaD == 7 ):
        print("CRAPS! Você perdeu")
        lancamento = 1
     elif(somaD == ponto):
        print("Parabéns, você ganhou!")
        lancamento = 1
     

    